Source/WebCore/rendering/RenderTextControlSingleLine.cpp

```diff
@@ -322,7 +322,7 @@ LayoutUnit RenderTextControlSingleLine::preferredContentLogicalWidth(float charW
     // of MS Shell Dlg, the default font for textareas in Firefox, Safari Win and
     // IE for some encodings (in IE, the default font is encoding specific).
     // 4027 is the (xMax - xMin) value in the "head" font table for MS Shell Dlg.
-    if (family == "Lucida Grande")
+    if (family == "Lucida Grande"_s)
```

Comment:
This code is super-obsolete. Lucida Grande hasn’t been the default font for anything for a long, long time.